tee-worker/ts-tests/indirect_calls.ts

```diff
@@ -38,21 +38,27 @@ export async function createIdentity(
     aesKey: HexString,
     listening: boolean,
     identity: LitentryIdentity
-): Promise<HexString[] | undefined> {
+): Promise<IdentityGenericEvent | undefined> {
     const encode = context.substrate.createType('LitentryIdentity', identity).toHex();
     const ciphertext = encryptWithTeeShieldingKey(context.teeShieldingKey, encode).toString('hex');
     const nonce = await context.substrate.rpc.system.accountNextIndex(signer.address);
     await context.substrate.tx.identityManagement
         .createIdentity(context.shard, signer.address, `0x${ciphertext}`, null)
         .signAndSend(signer, { nonce });
     if (listening) {
-        const event = await listenEncryptedEvents(context, aesKey, {
+        const identityEvent = await listenEncryptedEvents(context, aesKey, {
+            module: 'identityManagement',
+            method: 'identityCreated',
+            event: 'IdentityCreated',
+        });
+        const codeEvent = await listenEncryptedEvents(context, aesKey, {
```

Comment:
These events are in the same block，but the logic of the code is possible to listen to the event in different block, so i think it would be better to change the `listenEncryptedEvents`.